packages/test-utils/package.json

```diff
@@ -58,9 +58,10 @@
   ],
   "dependencies": {
     "@chainsafe/bls": "7.1.3",
-    "@chainsafe/bls-keystore": "^3.1.0",
-    "@lodestar/params": "^1.20.1",
-    "@lodestar/utils": "^1.20.1",
+    "@chainsafe/bls-keystore": "^3.0.1",
+    "@lodestar/params": "^1.20.2",
+    "@lodestar/utils": "^1.20.2",
+
```

Comment:
```suggestion
    "@chainsafe/bls-keystore": "^3.1.0",
    "@lodestar/params": "^1.20.2",
    "@lodestar/utils": "^1.20.2",
```

Reply:
or wait why is the bls-keystore line even modified :thinking:

Reply:
I just took the incoming change from the merge conflict. I thought we were ok with 3.0.1?

Reply:
ah now I get it, why did we merge unstable into stable, now all changes from unstable are in here

that needs to be reverted, we need to merge stable into unstable, not the other way around

Reply:
> I thought we were ok with 3.0.1?

should be hte version on stable, but if we merge it like this it would downgrade the version on unstable branch as well but we want 3.1.0 there